django/thunderstore/modpacks/api/experimental/tests/test_legacyprofile.py

```
@@ -107,3 +110,41 @@ def test_experimental_api_legacyprofile_retrieve(api_client: APIClient) -> None:
+        test_content = b"test profile data"
+
+        response = api_client.post(
+            reverse("api:experimental:legacyprofile.create"),
+            data=test_content,
+            content_type="application/octet-stream",
+        )
+
+        assert response.status_code == 200
+
+        while connection.run_on_commit:
+            sids, func = connection.run_on_commit.pop(0)
```

Ruff (0.14.13)

Unpacked variable `sids` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)